infer_segmentation.py

```diff
@@ -9,8 +9,8 @@
 import git
 from models import generate_compiled_segmentation_model
 
-
-metadata_file_name = 'metadata.yaml'
+# infer can be run multiple times (labels, overlay), create new metadata each time
+metadata_file_name = 'metadata_' + datetime.now(pytz.UTC).strftime('%Y%m%dT%H%M%SZ') + '.yaml'
```

Comment:
I remember us talking about this but now looking through the code am a bit confused -- if infer is run twice isn't each run stored in a completely new folder in infer?

Reply:
no new folder, based on experience, it will overwrite any files with same names and just add the new files (aka new metadata)

Reply:
ohhh right! because in the inference file name, for example: `8bit_AS4_S2_P1_L4_2560_1750_2160_segmentation-model-xlarge_esrf16_v2_HiLoadTrain_20191117T214257Z` the timestamp at the end is for the model. I think the picture in my head was inferences shouldn't be timestamped because inference of (model+dataset) should always be the same so no need to timestamp. I get it now, cool feel free to ignore this then!

Reply:
or if i run infer the first time for overlays, then after running the second time for labels only, the folder will have both over lays and labels and two metadata files